build/media_source/plg_system_guidedtours/js/guidedtours.es6.js

```diff
@@ -15,6 +15,71 @@ function emptyStorage() {
   sessionStorage.removeItem('tourId');
   sessionStorage.removeItem('tourToken');
   sessionStorage.removeItem('previousStepUrl');
+  sessionStorage.removeItem('skipTour');
+  sessionStorage.removeItem('autoTourId');
+}
+
+/**
+  Synchronize tour state for this user in his account/profile
```

Comment:
```suggestion
  Synchronize tour state for this user in their account/profile
```

please try to use gender neutral labuage